apps/web/src/components/notification-center/NotificationRenewal/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { useState, type ReactElement } from 'react'
+import { Alert, Box, Button, Typography } from '@mui/material'
+import useSafeInfo from '@/hooks/useSafeInfo'
+import CheckWallet from '@/components/common/CheckWallet'
+import { useNotificationsRenewal } from '@/components/settings/PushNotifications/hooks/useNotificationsRenewal'
+import { useIsNotificationsRenewalEnabled } from '@/components/settings/PushNotifications/hooks/useNotificationsTokenVersion'
+
+const NotificationRenewal = (): ReactElement => {
+  const { safe, safeLoaded } = useSafeInfo()
+  const [isRegistering, setIsRegistering] = useState(false)
+  const { renewNotifications, needsRenewal, numberChainsForRenewal } = useNotificationsRenewal()
+  const isNotificationsRenewalEnabled = useIsNotificationsRenewalEnabled()
+
+  if (!needsRenewal || !isNotificationsRenewalEnabled) {
+    // No need to renew any Safe's notifications
+    return <></>
+  }
+
+  const handeSignClick = async () => {
+    setIsRegistering(true)
+    await renewNotifications()
```

Comment:
Can this reject?

Reply:
I added error handling in `renewNotifications`, so it won't reject